schema/crdb/README.adoc

```diff
@@ -8,7 +8,11 @@ This directory describes the schema(s) used by CockroachDB.
 We use the following conventions:
 
 * `schema/crdb/VERSION/up.sql`: The necessary idempotent migrations to transition from the
-  previous version of CockroachDB to this version.
+  previous version of CockroachDB to this version. These migrations will always be placed
+  within one transaction per file.
+** If more than one change is needed per version, any number of files starting with `up`
```

Comment:
"More than one change" is easily misinterpreted — might be good to link to something that would tell you what needs to be in its own transaction. I guess it would be that very complicated CRDB doc. It might be cool if we had our own version of that doc that tried to distill the relevant bits into something reasonable — so far that might be stuck in PR comments.

Reply:
I expanded on this section, and added a link to the CRDB docs. I do still think it's difficult to get right, but hopefully this clarifies things more than before.